runtime/tuple_test.go

```diff
@@ -102,6 +102,19 @@ func TestTupleContains(t *testing.T) {
 	}
 }
 
+func TestTupleCount(t *testing.T) {
+	cases := []invokeTestCase{
+		{args: wrapArgs(NewTuple(), NewInt(1)), want: NewInt(0).ToObject()},
+		{args: wrapArgs(NewTuple(None, None, None), None), want: NewInt(3).ToObject()},
+		{args: wrapArgs(NewTuple()), wantExc: mustCreateException(TypeErrorType, "count() takes exactly one argument (1 given)")},
```

Comment:
This won't actually exercise the code that I'm concerned about that we're discussing above, so adding this test is probably not very useful.